raco/backends/myria/tests/test_myria_execution.py

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
It would be nice to make the connection part of an integration test_suite. I understand that it cannot be a unit test because it depends on a connection to myria.